package.json

```diff
@@ -47,7 +47,7 @@
     "ts-simple-ast": "0.74.0",
     "tslint": "4.4.2",
     "tslint-config-standard": "4.0.0",
-    "typescript": "2.6.2"
+    "typescript": "^2.7.1"
```

Comment:
It would be better to lock the version with `~` instead of `^`

Reply:
isn't pinned to `2.7.1` by `package-lock.json`?

Reply:
oh my, forgot about that :D

Reply:
eheh no problem (and thanks for reviewing btw)